node/src/types/block.rs

```diff
@@ -1034,7 +1280,41 @@ pub struct Block {
     body: BlockBody,
 }
 
+/// The hashing algorithm used for the header and the body of a block
+#[derive(PartialEq, Eq, Clone, Copy, Debug)]
+pub enum HashingAlgorithmVersion {
+    /// Version 1
+    V1,
+    /// Version 2
+    V2,
+}
+
+impl HashingAlgorithmVersion {
+    #[cfg(feature = "casper-mainnet")]
+    pub(crate) const HASH_V2_PROTOCOL_VERSION: ProtocolVersion =
+        ProtocolVersion::from_parts(1, 4, 0);
+
+    #[cfg(not(feature = "casper-mainnet"))]
+    pub(crate) const HASH_V2_PROTOCOL_VERSION: ProtocolVersion =
+        ProtocolVersion::from_parts(0, 0, 0);
```

Comment:
Seems like the `casper-mainnet` feature hasn't been added to the manifest.

Also, I'm not sure the hashing algorithm version should be defined by the protocol version.  It's probably fine to leave as is for now though, since we still need to move the `Block` to casper-types where it will end up with a hard-coded API version.

Reply:
Hmm - the manifest is only a part of it, there is also the part where it should be enabled during compilation etc., though I think we turned that off for now even on `feat-fast-sync` due to issues with the client. So, should I port only the feature definition in the manifest for now, or should we do something different? cc @afck @xcthulhu

Reply:
> since we still need to move the Block to casper-types where it will end up with a hard-coded API version.

@Fraser999 Do you mean ABI version?

> So, should I port only the feature definition in the manifest for now, or should we do something different?

I'm not sure.  Maybe a link to the area of the code we have in mind?